### ISSUE TITLE
make test failed

### ISSUE THREAD

**Issue by heysion** [zfsonlinux/zfs-test#3]
1 setp :
wget https://github.com/zfsonlinux/zfs-test/archive/master.zip
unzip zfs-test-master.zip
2 setp:
./autogen.sh & ./configure & make 
3 setp:
adduser --home /var/tmp/test_results --shell /usr/bin/ksh --gecos "ZFS Test Suite" --disabled-password zfs-tests
echo 'zfs-tests      ALL = (ALL:ALL) NOPASSWD: ALL' >> sudo /etc/sudoers
4 setp:
make test
........................
make[1]: Leaving directory '/home/xx/zfs-test-master/test/zfs-tests/cmd/file_check'
/bin/bash: /usr/src/zfs-test/scripts/mkfile: No such file or directory
/bin/bash: /usr/src/zfs-test/scripts/mkfile: No such file or directory
/bin/bash: /usr/src/zfs-test/scripts/mkfile: No such file or directory
ksh: /usr/src/zfs-test/test/zfs-tests/cmd/scripts/zfstest.ksh: not found
Makefile:929: recipe for target 'test' failed
make: *** [test] Error 127

**Comment by FransUrbo** [comment]
Yes. There's probably a lot more stuff like this. This is only the initial attempt.

You'll need to have extracted this in <code>/usr/src</code>.
